docs/README_API_INSTALL_PYTHON_LINUX.md

```diff
@@ -42,7 +42,7 @@ Now activate this new virtual environment for WeVoteServer:
 
 ## Installing Python 3 on Linux
 
-    (WeVoteServer) $ pip3 install django-toolbelt
```

Comment:
I think this package my be outdated. This script always gave me an error and I was able to get everything installed by just installing django